manifests/p/Pulumi/Pulumi/3.28.0/Pulumi.Pulumi.yaml

```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+
+PackageIdentifier: Pulumi.Pulumi
+PackageVersion: 3.28.0
+PackageLocale: en-US
+Publisher: Pulumi Corp

```

Comment:
```suggestion
Publisher: Pulumi
```
This must match Control Panel.

Reply:
Hi @ItzLevvie, can you please elaborate? You mean the publisher has to be same the name as the program itself?

Reply:
In the Windows OS, the Control Panel (a.k.a. Apps & features, Installed apps, Add or Remove Programs) is where you see all of your installed apps on your PC. It will show the name of the program, the version of the program, and the publisher of the program, as well as when it was installed on your PC and more.

Documentation in https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.1.0/singleton.md#fields requires the manifest to have the same publisher as in Control Panel.

![image](https://user-images.githubusercontent.com/11600822/161868772-6c5a356c-b900-41c8-8e46-123cc24bc4bc.png)
![image](https://user-images.githubusercontent.com/11600822/161869094-4b834ea7-f207-4fd5-9a9c-811639b96f5e.png)

It shows up as "Pulumi" in Control Panel because of the "Publisher" registry key.

Reply:
@ItzLevvie Thanks a lot for the detailed explanation and for checking where the issue is coming from 🙏 ❤️ 

I will make the change to make both publisher and manufacturer the same. However, since I don't have write access to this PR, would you like to close this and await a new "fixed" PR instead?